src/arcam/fmj/state.py

```diff
@@ -163,6 +180,20 @@ def get_incoming_audio_format(
             IncomingAudioConfig.from_int(value[1]),
         )
 
+    def get_incoming_audio_sample_rate(self) -> int:
+        value = self._state.get(CommandCodes.INCOMING_AUDIO_SAMPLERATE)
+        if value is None:
+            return 0
+        match value[0]:
```

Comment:
Im not sure ive looked the python revision for match support. Please add a lookup dict instead.